public/app/features/dashboard/components/ShareModal/SharePublicDashboard/SharePublicDashboardUtils.ts

```diff
@@ -37,6 +37,7 @@ export interface SessionDashboard {
 export interface SessionUser {
   email: string;
   firstSeenAtAge: string;
+  lastSeenAtAge: string;
```

Comment:
this is not optional. I imagine in the backend code you're always returning it and, otherwise, it's an empty string?

Reply:
It's never empty, the backend will return the value of `firstSeenAtAge` when `lastSeenAtAge` is empty or the table has no value for `lastSeenAt`